http_collector_client.go

```diff
@@ -0,0 +1,140 @@
+package lightstep
```

Comment:
This entire file's format seems a bit off to me on github, maybe run `go fmt` again?

Reply:
done!